shopfloor/models/stock_picking_batch.py

```diff
@@ -1,43 +0,0 @@
-from odoo import api, fields, models
-
-
-class StockPickingBatch(models.Model):
```

Comment:
@guewen hmm is this deletion wanted?
The other fields are required.

Reply:
https://travis-ci.com/github/camptocamp/wms/jobs/329457562#L1375

Reply:
restored the file :)